components/layout/blog-three-column/blog-three-column.hbs

```diff
@@ -0,0 +1,77 @@
+<div class="section--bg-white">
+    <div class="grid-col-content__three section__inner">
+
+        <a href="#" class="content-block--clickable content-block--blog">
+            <div class="content-block__upper-blog">
+                <picture class="post-image">
+                    <source media="(max-width: 799px)" srcset="https://www.fillmurray.com/g/355/180">
+                    <source media="(min-width: 800px)" srcset="https://www.fillmurray.com/g/268/160">
+                    <img class="content-block__image" src="https://www.fillmurray.com/g/268/160" role="presentation"
+                        alt="" />
+                    <source>
+                </picture>
+            </div><!-- .content-block__upper-blog -->
+            <div class="content-block__lower-blog">
+                <h3 class="content-block__title anchor-primary h6">
+                    Using your UX skills to architect your career.
+                </h3>
+                <div class="content-block__blog-author-date">
+                    <label for="source" class="content-block__author">By Mags Hanley</label>
+                    <time class="content-block__date-blog" datetime="2021-06-10"><span
+                            class="content-block__date-month-day-year" aria-label="June">06.02.21</span></time>
+                </div>
+                <p class="content-block__blog-text-tag">Managing People, Managing Freelancers</p>
+
+            </div><!-- .content-block__lower-blog -->
+        </a><!-- .content-block--clickable .content-block--blog -->
+
+
+        <a href="#" class="content-block--clickable content-block--blog">
+            <div class="content-block__upper-blog">
+                <picture class="post-image">
+                    <source media="(max-width: 799px)" srcset="https://www.fillmurray.com/g/355/460">
+                    <source media="(min-width: 800px)" srcset="https://www.fillmurray.com/g/268/392">
+                    <img class="content-block__image" src="https://www.fillmurray.com/g/268/392" role="presentation"
+                        alt="" />
+                    <source>
+                </picture>
+            </div><!-- .content-block__upper-blog -->
+            <div class="content-block__lower-blog">
+                <h3 class="content-block__title anchor-primary h6">
+                    Why Diversity Is Crucial for Decision Making, Creativity and the Success of Your Teams
+                </h3>
+                <div class="content-block__blog-author-date">
+                    <label for="source" class="content-block__author">By Craig Badings</label>
```

Comment:
Label elements should only ever be used in combination with inputs. Just use a span here.